config/initializers/active_model_serializers.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module ASMTrace
+  def serializable_hash(*)
+    if Rails.env.production?
+      Google::Cloud::Trace.in_span "active_model_serializers#serializable_hash##{self.class.name}" do |_span|
```

Comment:
If this doesn't work, we may need to [set the trace context first](https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-trace/lib/google/cloud/trace.rb#L242).

I assume Rails already sets one, so I'll try to track down to verify.

Reply:
It'd be inside the stackdriver gem (or something stackdriver requires). Though I assume it must since we're getting AR traces already...

Reply:
Yeah, stackdriver is just an empty thing that loads the google-cloud-trace gem's railtie. Digging thru the railtie to figure out what it does.